src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputFormattedWhitespaceAfterBad.java

```diff
@@ -0,0 +1,82 @@
+package com.google.checkstyle.test.chapter4formatting.rule462horizontalwhitespace;
+
+/** some javadoc. */
+public class InputFormattedWhitespaceAfterBad {
+  /** some javadoc. */
+  public void check1(int x, int y) {
+    // violation below ''for' construct must use '{}'s.'
+    for (int a = 1, b = 2; a < 5; a++, b--)
+      ;
```

Comment:
its this checkstyle of formatter defect ?

Reply:
formatter, will open an issue for it on their GH

Reply:
[4.1.1 Use of optional braces](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used):

> Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.

Reply:
@Zopsss , please open issue to let us merge this PR

Reply:
not a defect from our side, it is formatter's issue so it should not block this PR. We can merge this PR

Reply:
https://github.com/google/google-java-format/issues/51